docs/manual/index-checker.tex

```diff
@@ -457,8 +457,119 @@ \section{Binary search indices\label{index-searchindex}}
   This is the top type, and programmers should rarely need to write it.
 \end{description}
 
+\section{Typing the result of \<String.indexOf(String)>\label{index-indexof}}
+
+The methods \sunjavadoc{java/lang/String.html\#indexOf-java.lang.String-}{String.indexOf}
+and \sunjavadoc{java/lang/String.html\#lastIndexOf-java.lang.String-}{String.lastIndexOf} return an index
+of a given substring within a given string, or -1 if no such substring exists.
+The index \<i> returned from \<receiver.indexOf(substring)> satisfies this property:
+\code{i==-1 || i>=0 \&\& i<=receiver.length() - substring.length()}.
```

Comment:
I'd prefer to use spaces between operators (so `i == -1` instead of `i==-1`, etc.). This makes the code more readable.

Reply:
This appears a few other times below, as well.